rdbms/src/main/java/org/spine3/server/storage/jdbc/query/DeleteRowQuery.java

```diff
@@ -0,0 +1,129 @@
+import org.spine3.server.storage.jdbc.util.IdColumn;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static java.lang.String.format;
+import static org.spine3.base.Stringifiers.idToString;
+import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.EQUAL;
+import static org.spine3.server.storage.jdbc.Sql.Query.DELETE_FROM;
+import static org.spine3.server.storage.jdbc.Sql.Query.PLACEHOLDER;
+import static org.spine3.server.storage.jdbc.Sql.Query.WHERE;
+
+/**
+ * A query for deleting one or many items by a id of a given column.
+ *
+ * @author Dmytro Dashenkov.
+ */
+public class DeleteRowQuery<I> extends StorageQuery {
+
+    private static final String FORMAT_PLACEHOLDER = "%s";
+
+    private static final String TEMPLATE =
+            DELETE_FROM + FORMAT_PLACEHOLDER + WHERE + FORMAT_PLACEHOLDER + EQUAL + PLACEHOLDER;
+
+    private static final int COLUMN_VALUE_PARAM_INDEX = 1;
+
+    private final I id;
+    private final IdColumn<I> idColumn;
+
+    protected DeleteRowQuery(Builder<I> builder) {
+        super(builder);
+        this.id = builder.columnValue;
+        this.idColumn = builder.idColumn;
+    }
+
+    /**
+     * Executes the {@code DELETE} SQL statement.
+     *
+     * @return {@code true} if at least one row was deleted, {@code false} otherwise
+     */
+    public boolean execute() {
+        try (ConnectionWrapper connection = getConnection(false)) {
+            final PreparedStatement statement = prepareStatement(connection);
+            idColumn.setId(COLUMN_VALUE_PARAM_INDEX, id, statement);
+            final int rowsAffected = statement.executeUpdate();
+            connection.commit();
+            final boolean result = rowsAffected != 0;
+            return result;
+        } catch (SQLException e) {
+            throw new IllegalStateException(e);
+        }
+    }
+
+    public static <V> Builder<V> newBuilder() {
+        return new Builder<>();
+    }
+
+    public static class Builder<I> extends StorageQuery.Builder<Builder<I>, DeleteRowQuery> {
+
+        private String column;
+        private I columnValue;
+        private String table;
+        private IdColumn<I> idColumn;
+
+        public Builder<I> setIdColumnName(String column) {
+            this.column = checkNotNull(column);
+            return getThis();
+        }
+
+        public Builder<I> setIdValue(I value) {
+            this.columnValue = checkNotNull(value);
+            return getThis();
+        }
+
+        public Builder<I> setTableName(String table) {
+            this.table = checkNotNull(table);
+            return getThis();
+        }
+
+        private String composeSql() {
+            return format(TEMPLATE, table, column, idToString(columnValue));
+        }
+
+        @Override
+        public DeleteRowQuery<I> build() {
+            checkNotNull(column, "ID column name must be set first");
```

Comment:
Please remove `first `from these messages. Having some many of them in a row makes no sense: there is no `first` anymore.

And also, it's a formal message, not a console app hint.